docs/src/pages/components/chatbot/index.page.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Chatbot
+description: A simple way to add a conversational UI into your app is to use our ChatBot Component.
+---
+
+import { Fragment } from '@/components/Fragment';
+import { Alert } from '@aws-amplify/ui-react';
+
+<Alert variation="info" heading="Legacy component">
+  This component has been migrated from the previous version of Amplify UI, but
+  has not been updated in this new version.
+</Alert>
```

Comment:
I'm a bit remorseful by how often we consider things "Legacy".  Since these are considered working versions, should we instead just indicate that we intend to create native ports?

```suggestion
<Alert variation="info" heading="Web Component">
	This component has not yet been converted to a native implementation.
</Alert>
```

Or should we bother saying anything at all and instead only mention the legacy aspect _when we have a `@next` version to use_?

Reply:
Yea that's a good point. I like your version it is less negative.